app/models/user.rb

```diff
@@ -100,11 +113,10 @@ def validate_pin_pattern
   # a success/failure response.
   # Accepts a response from the microservice, and returns.
   def send_request_to_patron_creator_service
-    Rails.logger.debug("send_user_to_patron_creator_service: start")
     query = {
       'names' => [last_name.upcase + ', ' + first_name.upcase],
       'emails' => [email],
-      'pin' => pin,
+      'pin' => '1234',
```

Comment:
Is this hardcoded change in here accidentally?

Reply:
Yes! Good catch.